arkouda/pdarrayclass.py

```diff
@@ -2622,13 +2622,78 @@ def rotr(x, rot) -> pdarray:
 
 
 @typechecked
-def power(pda: pdarray, pwr: Union[int, float, pdarray]) -> pdarray:
-    return pda**pwr
+def power(pda: pdarray, pwr: Union[int, float, pdarray], where: Union[bool, pdarray] = True) -> pdarray:
+    """
+    Raises an input value or array to a power, can be subjected to a condition that if True will raise value to desiered
```

Comment:
- `power` only takes an array, so I think we should remove the `input value` from this description
- typo `desiered` -> `desired`